lighthouse-core/gather/gatherers/console-messages.js

```diff
@@ -30,10 +30,6 @@ function remoteObjectToString(obj) {
   return `[${type} ${className}]`;
 }
 
-/**
- * @implements {LH.Gatherer.GathererInstance}
- * @implements {LH.Gatherer.FRGathererInstance}
```

Comment:
This seemed unnecessary.

Reply:
Depends on how explicit we want to be with knowing whether each gatherer supports FR and legacy or not :)

I erred on the side of "it's not yet obvious enough that a `require('../../fraggle-rock/gather/base-gatherer.js');` at the top means this gatherer supports both APIs", but if everyone on the team feels it's already clear, wfm 👍

Reply:
I would prefer keeping this around to keep it more explicit. The `@implements` was helpful when reviewing.

Reply:
`const FRGatherer = require('../../fraggle-rock/gather/base-gatherer.js');`

`class ... extends FRGatherer`

?

Reply:
That works for me.

One possible issue is that I already knew `FRGatherer` supports legacy mode. Probably not a big deal, I was mostly worried about missing the `require('../../fraggle-rock/gather/base-gatherer.js');` at the top of the file.